src/main/resources/lib/configFile/services/bean.js

```diff
@@ -0,0 +1,46 @@
+// Java-interface methods, service-ified for mocking
+
+
+function required(params, name) {
+    var value = params[name];
+    if (value === undefined) {
+        throw "Parameter '" + name + "' is required";
+    }
+    return value;
+}
+
+function nullOrValue(value) {
+    return value == null ? null : value;
+}
```

Comment:
I'm suprised we don't have a lib for this. 
I see this used left and right inside content studio and all places using js -> java